README.md

```diff
@@ -1,7 +1,7 @@
 <img src=http://storage.mds.yandex.net/get-devtools-opensource/250854/catboost-logo.png width=300/>
 
-[Documentation](https://tech.yandex.com/catboost/doc/dg/concepts/about-docpage/) |
-[Installation](https://tech.yandex.com/catboost/doc/dg/concepts/cli-installation-docpage/)
+[Documentation](https://tech.yandex.com/catboost/doc/dg/concepts/about-docpage/) | [Python Installation](https://tech.yandex.com/catboost/doc/dg/concepts/python-installation-docpage/) |
+[R Installation](https://tech.yandex.com/catboost/doc/dg/concepts/cli-installation-docpage/)
```

Comment:
This is not a link to R installation, it's about command line binary. Proper link is https://tech.yandex.com/catboost/doc/dg/concepts/r-installation-docpage/

Links to python/R installation also present in this README (lines 14-15). May be, it will be clearer if "Installation" link will be removed from the top of file?